authLdap.php

```diff
@@ -255,7 +257,14 @@ function authLdap_login($user, $username, $password, $already_md5 = false)
                 authLdap_debug('could not get user attributes from LDAP');
                 throw new UnexpectedValueException('dn has not been returned');
             }
+            if (! isset($attribs[0]['uid'][0])) {
```

Comment:
When we set the ```RealUID``` as option it can be any attribute. Therefore we'd have to replace the ```uid```here with ```$authLDAPRealUID```